### PR TITLE
Add Syklo info to Celo dapp README

### PR DESCRIPTION
**Pull request by crmanrique**
### Description
Celo Camp update on dapp README

### CODE REVIEW

File: packages/docs/developer-resources/celo-dapp-gallery.md [hunk 1]
```diff
@@ -56,6 +56,21 @@ At Dunia Payment, we are building an electronic wallet that will let people in S
...
+### Syklo by Air Protocol 
+
+Syklo is the first peer-to-peer mobile application, on [Air Protocol](https://www.airprotocol.org), that allows users to send remittances from anywhere in the world. It also serves as a digital wallet to store USDs thus enabling users to save in a stable currency.
+
+Through its use of the Celo blockchain, implemented on Air Protocol, it creates dynamic multisig escrows (using a threshold of 2/3). The escrows enable users to be confident their crypto payment is safe while sending their off-chain (P2P) transaction, such as sending USD via PayPal. If the exchange goes smoothly then the escrow is released, however if something goes sideways, then a mediation team can give a ruling and sign the release of the escrow to the appropriate user.
+
+This app is one example of what developers can achieve by using Celo and the Air Protocol. Air Protocol provides the infrastructure to create decentralized P2P transactions and Celo provides the blockchain to exchange and store the crypto assets safely.
+
+Air Protocol Github: https://github.com/air-protocol/nitrogen 
+To download Syklo Android apk click [here](https://www.airprotocol.org/syklo-v1.apk)
```
Comment (timmoreton) [inline packages/docs/developer-resources/celo-dapp-gallery.md]: nit: you'll need blank lines between these paragraphs or they'll all come out together.

File: packages/docs/developer-resources/celo-dapp-gallery.md [hunk 2]
```diff
@@ -56,6 +56,21 @@ At Dunia Payment, we are building an electronic wallet that will let people in S
...
+
+This app is one example of what developers can achieve by using Celo and the Air Protocol. Air Protocol provides the infrastructure to create decentralized P2P transactions and Celo provides the blockchain to exchange and store the crypto assets safely.
+
+Air Protocol Github: https://github.com/air-protocol/nitrogen 
+To download Syklo Android apk click [here](https://www.airprotocol.org/syklo-v1.apk) 
+For Syklo iOS app, we'll be sharing through TestFlight, so please share your TestFlight account [here](https://forms.gle/1oLc9S8n71bK7sTi6) 
+Coming soon! We'll open source Syklo code too, so anyone can use it or change it for their needs :) 
+
+Note: To be able to accept offers in "Market", please fund your account (you can find in on the menu -> Wallet) by using the following link: https://celo.org/developers/faucet or wait for someone to accept your transaction after making a proposal of "Add funds" (menu -> Add funds)
```
Comment (timmoreton) [inline packages/docs/developer-resources/celo-dapp-gallery.md]: please add markdown for this link

Comment (timmoreton) [inline packages/docs/developer-resources/celo-dapp-gallery.md]: typo: missing `in` -> `it`

**Review by timmoreton — COMMENTED** [review]
Thanks for submitting this! few minor comments

File: packages/docs/developer-resources/celo-dapp-gallery.md (hunk 1 above)
Comment (crmanrique) [reply packages/docs/developer-resources/celo-dapp-gallery.md]: I agree, I made the change.

File: packages/docs/developer-resources/celo-dapp-gallery.md (hunk 2 above)
Comment (crmanrique) [reply packages/docs/developer-resources/celo-dapp-gallery.md]: thanks! didn't saw the typo

File: packages/docs/developer-resources/celo-dapp-gallery.md [hunk 3]
```diff
@@ -56,6 +56,23 @@ At Dunia Payment, we are building an electronic wallet that will let people in S
...
+
+Syklo is the first peer-to-peer mobile application, on [Air Protocol](https://www.airprotocol.org), that allows users to send remittances from anywhere in the world. It also serves as a digital wallet to store USDs thus enabling users to save in a stable currency.
+
+Through its use of the Celo blockchain, implemented on Air Protocol, it creates dynamic multisig escrows (using a threshold of 2/3). The escrows enable users to be confident their crypto payment is safe while sending their off-chain (P2P) transaction, such as sending USD via PayPal. If the exchange goes smoothly then the escrow is released, however if something goes sideways, then a mediation team can give a ruling and sign the release of the escrow to the appropriate user.
+
+This app is one example of what developers can achieve by using Celo and the Air Protocol. Air Protocol provides the infrastructure to create decentralized P2P transactions and Celo provides the blockchain to exchange and store the crypto assets safely.
+
+Air Protocol Github: https://github.com/air-protocol/nitrogen 
+
+To download Syklo Android apk click [here](https://www.airprotocol.org/syklo-v1.apk)
```
Comment (barbaraliau) [inline packages/docs/developer-resources/celo-dapp-gallery.md:69]: Please add a `.` to the end of this sentence.

File: packages/docs/developer-resources/celo-dapp-gallery.md [hunk 4]
```diff
@@ -56,6 +56,23 @@ At Dunia Payment, we are building an electronic wallet that will let people in S
...
+For Syklo iOS app, we'll be sharing through TestFlight, so please share your TestFlight account [here](https://forms.gle/1oLc9S8n71bK7sTi6)
```
Comment (barbaraliau) [inline packages/docs/developer-resources/celo-dapp-gallery.md:70]: `.` at end of this sentence

File: packages/docs/developer-resources/celo-dapp-gallery.md [hunk 5]
```diff
@@ -56,6 +56,23 @@ At Dunia Payment, we are building an electronic wallet that will let people in S
...
+This app is one example of what developers can achieve by using Celo and the Air Protocol. Air Protocol provides the infrastructure to create decentralized P2P transactions and Celo provides the blockchain to exchange and store the crypto assets safely.
+
+Air Protocol Github: https://github.com/air-protocol/nitrogen 
+
+To download Syklo Android apk click [here](https://www.airprotocol.org/syklo-v1.apk)
+For Syklo iOS app, we'll be sharing through TestFlight, so please share your TestFlight account [here](https://forms.gle/1oLc9S8n71bK7sTi6) 
+
+Coming soon! We'll open source Syklo code too, so anyone can use it or change it for their needs :) 
+
+Note: To be able to accept offers in "Market", please fund your account (you can find it on the menu -> Wallet) by using the following [link](https://celo.org/developers/faucet) or wait for someone to accept your transaction after making a proposal of "Add funds" (menu -> Add funds)
```
Comment (barbaraliau) [inline packages/docs/developer-resources/celo-dapp-gallery.md:74]: `.` at end of this sentence.

**Review by barbaraliau — CHANGES_REQUESTED** [review]
Update the branch and fix some small punctuation changes, and then this is good to be merged. Thanks for adding this!

**Pull request closed without merging**